tests/lib/ansible_helper.py

```diff
@@ -153,14 +152,8 @@ def run_play_raw(self, playbook):
         ))
         logger.info(f'Running playbook {path}')
         try:
-            subprocess.run(
-                ['ansible-playbook', '-i', self.inventory_dir, path],
-                check=True,
-                env={
-                    'SSH_AUTH_SOCK': self.workspace.ssh_agent_auth_sock,
-                    'SSH_AGENT_PID': self.workspace.ssh_agent_pid,
-                }
-            )
+            self.workspace.execute(
```

Comment:
You are dropping the `check=True` here.

Reply:
execute defaults to check=True now (since we use it more often then not)